c7n/resources/rdscluster.py

```diff
@@ -54,27 +54,43 @@ class Meta(object):
 class Delete(BaseAction):
 
     schema = type_schema(
-        'delete', **{'skip-snapshot': {'type': 'boolean'}})
+        'delete', **{'skip-snapshot': {'type': 'boolean'},
+                     'delete-instances': {'type': 'boolean'}})
 
-    def process(self, resources):
-        self.skip = self.data.get('skip-snapshot', False)
+    def process(self, clusters):
+        skip = self.data.get('skip-snapshot', False)
+        delete_instances = self.data.get('delete-instances', False)
```

Comment:
if we do need this feels like we should do it post deletion of the cluster itself, else how would you get a final snapshot if you have no instances.